src/components/dashboard/Dashboard.js

```diff
@@ -750,16 +755,17 @@ const Dashboard = props => {
       <FeedList
         data={feedRef.current}
         handleFeedSelection={handleFeedSelection}
-        initialNumToRender={10}
+        initialNumToRender={15}
```

Comment:
Add short comment please abo it what’s solved by adjusting those props

Reply:
Will do

Reply:
done